apps/editing-toolkit/editing-toolkit-plugin/wpcom-domain-upsell-callout/class-wpcom-domain-upsell-callout.php

```diff
@@ -57,6 +57,7 @@ public function enqueue_script_and_style() {
 			$version,
 			true
 		);
+		wp_script_add_data( 'wpcom-domain-upsell-callout-script', 'strategy', 'defer' );
```

Comment:
It's still works but I not sure how to ensure it becomes better and if it's compatible with `wp_enqueue_script`. Asked more about it here: pdDR7T-12w-p2#comment-1240

Reply:
We got a response: pdDR7T-12w-p2#comment-1241

Reply:
It's loading with defer:
```
<script src='https://mywordpresssite.com/wp-content/plugins/editing-toolkit-plugin/dev/wpcom-domain-upsell-callout/dist/wpcom-domain-upsell-callout.min.js?m=1698086731i&#038;ver=3c21936d7c6248ed824a' id='wpcom-domain-upsell-callout-script-js' defer data-wp-strategy='defer'></script>
```

Reply:
Thanks for checking it. It's working as expected. Nice work @lupus2k!